src/util/piped_process.cpp

```diff
@@ -102,19 +102,19 @@
 #  define BUFSIZE 2048
 
 #ifdef _WIN32
-std::wstring process_windows_args(const std::vector<std::string> commandvec)
+std::wstring process_windows_args(const std::vector<std::string> *commandvec)
```

Comment:
:no_entry_sign: Adding that `*` passes by pointer, not by reference. To pass by reference you should use `&`. Passing by pointer is an inferior solution as it allows `nullptr` to be passed as an argument, whereas a reference is non-nullable (as is passing by value). This issue should be fixed before this PR is merged, because it affects the way the constructor is used. So if we leave resolving it later then we may need to fix additional usages of the constructor built on top of this PR.

Reply:
Looks to be fixed.